explainer.md

```diff
@@ -1,22 +1,26 @@
 ## Explainer
 
-Author: Eric Willigers &lt;<ericwilligers@chromium.org>&gt;<br>
-Author: Jay Harris &lt;<harrisjay@chromium.org>&gt;<br>
-Author: Raymes Khoury &lt;<raymes@chromium.org>&gt;
+Authors: 
+* Eric Willigers &lt;<ericwilligers@chromium.org>&gt;<br>
+* Jay Harris &lt;<harrisjay@chromium.org>&gt;<br>
+* Darwin Huang &lt;<huangdarwin@chromium.org>&gt;<br>
+* Raymes Khoury &lt;<raymes@chromium.org>&gt;
 
 ### Motivation
 
-This proposal gives web applications a way to register their ability to handle (read, stream, edit) files with given MIME types and/or file extensions.
+This proposal gives web applications a way to register their ability to handle (read, stream, edit) files with given MIME types and/or file extensions. This then allows an operating system's file manager or other native UX to select a PWA to handle the file.
```

Comment:
How about?:

...`to handle the file using a PWA similar to how it would handle the file using some other associated native app.`